services/wallet/router/router_v2.go

```diff
@@ -529,30 +528,19 @@ func (r *Router) SuggestedRoutesV2(ctx context.Context, input *RouteInputParams)
 						l1FeeWei, _ = r.feesManager.GetL1Fee(ctx, network.ChainID, txInputData)
 					}
 
-					baseFee, err := r.feesManager.getBaseFee(ctx, client)
+					fees, err := r.feesManager.SuggestedFees(ctx, network.ChainID)
 					if err != nil {
 						continue
 					}
 
-					priorityFees, err := r.feesManager.getPriorityFees(ctx, client, baseFee)
-					if err != nil {
-						continue
-					}
-					selctedPriorityFee := priorityFees.Medium
-					if input.GasFeeMode == GasFeeHigh {
-						selctedPriorityFee = priorityFees.High
-					} else if input.GasFeeMode == GasFeeLow {
-						selctedPriorityFee = priorityFees.Low
-					}
-
 					amountOut, err := pProcessor.CalculateAmountOut(processorInputParams)
 					if err != nil {
 						continue
 					}
 
-					maxFeesPerGas := new(big.Float)
-					maxFeesPerGas.Add(new(big.Float).SetInt(baseFee), new(big.Float).SetInt(selctedPriorityFee))
-					estimatedTime := r.feesManager.TransactionEstimatedTime(ctx, network.ChainID, maxFeesPerGas)
+					maxFeesPerGas := fees.feeFor(GasFeeMedium)
```

Comment:
Can we use `input.GasFeeMode` here instead of hardcoding it to `GasFeeMedium` on statusgo side?